src/somef/utils/constants.py

```diff
@@ -183,6 +220,10 @@
 
 # Spdx url
 SPDX_BASE = "https://spdx.org/licenses/"
+
+# Development status. 
+DEVELOPMENT_STATUS = "developmentStatus"
```

Comment:
Why do we have this? There is a CAT_STATUS already in place. That's how it's called in somef.